src/main/resources/templates/customer/create.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="/common.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR&display=swap" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
+          integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4"
+          crossorigin="anonymous"></script>
+  <style>
+    input[type="submit"] {
+      width: 100%;
+      margin-top: 10px;
+    }
+
+  </style>
+</head>
+<body>
+<div class="header3">고객 입력</div>
+<div>
+  <form action="/view/customer/save" method="post">
+    <div class="inputHeader">고객 이름</div>
+    <div>
+      <div class="mb-3">
+        <input type="text" name="customerName" class="form-control" placeholder="이름" required>
+      </div>
+    </div>
+    <div class="inputHeader">고객 이메일</div>
+    <div>
+      <div class="mb-3">
+        <input type="email" name="customerEmail" class="form-control" placeholder="name@example.com" required>
+      </div>
+    </div>
+    <div class="inputHeader">고객 타입</div>
+    <div>
+      <select name="customerType" class="form-select" aria-label=".form-select-lg example" required>
+        <option selected>고객 타입을 선택해주세요.</option>
+        <option value="NORMAL">normal</option>
+        <option value="BLACKLIST">blacklist</option>
+      </select>
+    </div>
+    <input type="submit" class="btn btn-outline-success" value="고객 생성"/>
+  </form>
+  <button style="width:100%; margin-top:10px;" type="button" onclick="javascript:location.href='/view/customer/'" class="btn btn-secondary">돌아가기</button>
```

Comment:
rest api 규칙이긴 한데... 
url 마지막에는 슬래시를 포함시키지 말아주세요!
(규칙 관련해서는 rest api PR에서 빡세게 보도록하겠습니다)

```suggestion
  <button style="width:100%; margin-top:10px;" type="button" onclick="javascript:location.href='/view/customer'" class="btn btn-secondary">돌아가기</button>
```